vcd/resource_vcd_network_routed.go

```diff
@@ -139,12 +144,20 @@ func resourceVcdNetworkRouted() *schema.Resource {
 						},
 					},
 				},
-				Set: resourceVcdNetworkIPAddressHash,
 			},
 		},
 	}
 }
 
+// suppressAlways suppresses the processing of the property unconditionally
+// Used when we want to remove a property that should not have been
+// added in the first place, but we want to keep compatibility
+func suppressAlways() schema.SchemaDiffSuppressFunc {
```

Comment:
We have seperate supressfunc.go for such funtions.

Reply:
moved